components/hab/src/exec.rs

```diff
@@ -112,8 +112,7 @@ where
 /// Determine the channel from which to install Habitat-specific
 /// packages.
 fn internal_tooling_channel() -> ChannelIdent {
-    match ChannelIdent::from_env_var(INTERNAL_TOOLING_CHANNEL_ENVVAR) {
-        Ok(channel) => channel,
-        Err(_) => ChannelIdent::stable(),
-    }
+    hcore::env::var(INTERNAL_TOOLING_CHANNEL_ENVVAR)
+        .map(ChannelIdent::from)
+        .unwrap_or_else(|_| ChannelIdent::stable()) // or default or configured_value?
```

Comment:
I think default (i.e. "stable") is correct here, given that there's a separate variable for this particular channel.